app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/DataCreationActivity.kt

```diff
@@ -8,6 +8,7 @@ import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 
 import com.github.HumanLearning2021.HumanLearningApp.databinding.ActivityDataCreationBinding
+import com.github.HumanLearning2021.HumanLearningApp.model.Category
```

Comment:
Since we are using this we should maybe remove the other category class in " com.github.HumanLearning2021.HumanLearningApp" that was in my previous merge